wagtail/admin/templates/wagtailadmin/icons/comment.svg

```diff
@@ -1,4 +1,4 @@
-<symbol id="icon-comment" viewBox="0 0 30 30">
-    <!-- Font Awesome Free 5.15.3 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) -->
-    <path d="M15 8c-4.438 0-8 2.938-8 6.5 0 1.563.656 2.969 1.781 4.094-.406 1.593-1.719 2.968-1.719 3a.3.3 0 00-.062.281c.063.094.125.125.25.125 2.063 0 3.625-.969 4.375-1.594A9.81 9.81 0 0015 21c4.406 0 8-2.906 8-6.5 0-3.563-3.594-6.5-8-6.5zm-4 7.5c-.563 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1zm4 0c-.563 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1zm4 0c-.563 0-1-.438-1-1 0-.531.438-1 1-1 .531 0 1 .469 1 1 0 .563-.469 1-1 1z"></path>
+<symbol id="icon-comment" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512">
```

Comment:
note: #8222 changes all icons from `<symbol>...</symbol>` to `<svg>..</svg>` so either this will need updating if #8222 gets in first, or the other way around

Reply:
```suggestion
<symbol id="icon-comment" viewBox="0 0 512 512">
```